### DISCUSSION TITLE
Component library monorepo using panda css

### DISCUSSION THREAD

**Discussion by AnteVukusic** [chakra-ui/panda#2597] [Q&A]
Hello, I was wondering is there an example library built using rollup which uses panda CSS? I'm creating a monorepo library which will have under packages core, modules and I assume styled-system package in which panda's style-system is build (according to Panda css docs). But I'm having issues with way of working. My components will be polymorphic and should be exported as "StyledComponent" from panda, also end user should be able to import {css} and other functions from my library. How should i achieve this, should I ship built panda files to my package or just ship panda config. If only shipping config I don't know at what point panda files are built on end user side?

Cheers

**Comment by astahmer** [comment]
this example uses tsup but the same thing should be applicable to rollup
https://github.com/astahmer/panda-monorepo-setup/tree/main/examples/with-import-map-package
